ega-charts/localega/templates/doa-deploy.yaml

```diff
@@ -42,7 +42,7 @@ spec:
         - name: SPRING_PROFILES_ACTIVE
           value: no-oss,LocalEGA
         - name: DB_INSTANCE
-          value: "{{- if .Values.filedatabase.deploy }}{{ template "localega.name" . }}-filedatabase:{{ .Values.filedatabase.servicePort }}{{- else }}{{ .Values.config.filedatabase_host }}:{{ .Values.config.filedatabase_port }}{{- end }}"
+          value: {{ .Values.config.postgres_host | quote }}
```

Comment:
This requires that `postgres_host` is defined.
The workaroud is to use `"{{- if .Values.config.postgres_host }}{{ .Values.config.postgres_host }}{{- else }}{{ template "localega.name" . }}-db{{- end }}"`